api/graphql/schema.graphql

```diff
@@ -1718,6 +1774,22 @@ type Mutation {
     @delete
     @guard
     @can(ability: "update", find: "id")
+  createAssessmentResult(
+    createAssessmentResult: CreateAssessmentResultInput! @spread
+  ): AssessmentResult @create @guard @can(ability: "create", injectArgs: true)
+  updateAssessmentResult(
+    id: UUID!
+    updateAssessmentResult: UpdateAssessmentResultInput! @spread
+  ): AssessmentResult
+    @validator(
+      class: "App\\GraphQL\\Validators\\UpdateAssessmentResultValidator"
```

Comment:
Given this needs the id of the updated model to check against the database. This validator is located elsewhere and also named differently from `CreateAssessmentResultInputValidator` as it doesn't just act on an input object.

Reply:
Actually, nevermind. Don't need to split out id from the input type for any reason it seems
https://github.com/GCTC-NTGC/gc-digital-talent/pull/8302/commits/7a3bc09457af30fcbb8dbbbeece60843d03f5970